back-end-side/Controllers/UploadController.cs

```diff
@@ -14,20 +14,35 @@ namespace back_end_side.Controllers
     
     public class UploadController : ControllerBase
     {
+        public static readonly List<Record> RecordsFromFile = new List<Record>();
+
         [HttpPost]
         public ActionResult Post([FromForm] FileModel file)
         {
+            
+            Console.WriteLine("BANK: " + file.Bank);
```

Comment:
No need?

Reply:
I forgot that it was there, will delete it